Jagusti/fvtt-wfrp4e-gmtoolkit
# Pause for players when running Session Turnover #182

## Type of change

## Checklist:
- [x] I have run linting on my code to follow the follow the [style](https://github.com/Jagusti/fvtt-wfrp4e-gmtoolkit/blob/dev/.eslintrc.json) of this project.
- [x] My changes generate no new unhandled or unexpected warnings.
- [x] My change requires an update to the [documentation](https://github.com/Jagusti/fvtt-wfrp4e-gmtoolkit/wiki).

## Related Issue(s) 
Fixes or addresses #issue(s): #182

## Description

### Summary of changes
Adds `push = true` parameter to togglePause()

### How has this been tested?

1. Log in separately as GM and player
2. Ensure game is unpaused.
3. Execute macro 'Session Turnover'
4. See game pauses for GM and player.

### Development / Testing Environment
- Foundry VTT: 10.285
- WFRP4e System: 6.1.2
- GM Toolkit:  6.0.0
- Macro (if applicable): Session Turnover 0.9.5 > 6.0.1
